velox/docs/monthly-updates/june-2022.rst

```diff
@@ -0,0 +1,61 @@
+********************
+June 2022 Update
+********************
+
+Documentation
+=============
+
+* Add documentation for :ref:`complex types writers<outputs-write>`.
+
+Core Library
+============
+
+* Add support for IntervalDayTime type.
+* Allow cast between Date type and Timestamp type.
+* Add fast path to :func:`zip` when all Arrays are flat and the same size.
+* Add GroupId operator.
+* Support Decimal type signature and type calculation using flex/bison.
+* Add NegatedBigintValuesUsingBitmask and NegatedBigintValuesUsingHashTable for Bigint types.
+* Add support for running tasks on the caller's thread.
+* Add limited ORC reader support.
+* Support conversion of DuckDB Decimals to Velox Decimals, and Decimal constant null vector.
+* Allow cast from JSON type to scalar types, Array type, and Map type.
+* Allow running join queries on the caller's thread.
+* Add q3 to TPC-H benchmark.
+* Separate the user exception stack from the runtime exception stack trace collection control.
+
+Presto Functions
+================
+
+* Extend :func:`plus`, :func:`minus` functions to support Date, IntervalDayTime argument types.
```

Comment:
Date, IntervalDayTime argument types -> DATE and INTERVAL DAY TO SECOND types.

Let use the following order: Add for scalar functions, Add for aggregate functions, followed by enhancements for scalar and aggregate functions. I.e. move this to the end.